crates/store/re_types/src/blueprint/components/component_column_selector_ext.rs

```diff
@@ -2,7 +2,7 @@ use re_log_types::EntityPath;
 
 impl super::ComponentColumnSelector {
     /// Create a [`Self`] from an [`EntityPath`] and a [`re_types_core::ComponentName`] expressed as string.
-    pub fn new(entity_path: &EntityPath, component_name: &str) -> Self {
+    pub fn new(entity_path: &EntityPath, component_name: String) -> Self {
```

Comment:
small question & suggestion: do we want to the parameters to be like this in the future? (fwiw `blueprint::datatypes::ComponentColumnSelector`'s field is called `component` already)
```suggestion
    pub fn new(entity_path: &EntityPath, component: String) -> Self {
        crate::blueprint::datatypes::ComponentColumnSelector::new(entity_path, component)
```

arguably in this case `column_name` would be even more more accurate?

Reply:
Ah good catch, guess I should stop and think from time to time when doing a mechanic refactor 😅.